client/src/locales/en-US.json

```
@@ -646,6 +646,8 @@
             "actionHistory": "History",
             "actionShare": "Sharing and roles",
             "actionDetails": "Details",
+            "actionUnMount": "Hide this workspace",
+            "actionMount": "Show this workspace",
```

IMO actions should be `Hide`/`Show`. The fact that we choose to mount/unmount on those actions is secondary (and only relevant on desktop, not on web) 
```suggestion
            "actionHide": "Hide this workspace",
            "actionShow": "Show this workspace",
```